src/caf/toolkit/io.py

```diff
@@ -229,3 +230,73 @@ def read_csv_matrix(
             matrix = matrix.reindex(matrix.columns, axis=0)
 
     return matrix
+
+
+def find_file_with_name(
```

Comment:
I feel an example could be useful to explain the usage of this function.

I.e. a file is stored with suffix x or y, we don't know which. This helps us find it.
It's a bit of a weird one because this is a by-product of normits weirdness, right?